File: Police_Recruits.py
def untreated_crimes():
    n = int(input())
    events = list(map(int, input().split()))
    sum_events = 0
    counter = 0

    for event in events:
        sum_events += event
        if sum_events < 0:
            counter += 1
            sum_events = 0

    print(counter)

if __name__ == "__main__":
    untreated_crimes()

File: Police_Recruits.cpp
#include <iostream>
using namespace std;

void untreated_crimes()
{
    int n;
    cin >> n;
    int events[n];
    int sum = 0;
    int counter = 0;
    for (int i = 0; i < n; i++)
    {
        cin >> events[i];
        sum += events[i];
        if (sum < 0)
        {
            counter++;
            sum = 0;
        }
    }
    cout << counter;
}

int main(){
    untreated_crimes();
    return 0;
}